pkg/webhookconfig/configmanager.go

```diff
@@ -733,7 +733,7 @@ func (m *webhookConfigManager) mergeWebhook(dst *webhook, policy *kyverno.Cluste
 	for _, gvr := range gvrList {
 		groups = append(groups, gvr.Group)
 		versions = append(versions, gvr.Version)
-		rsrcs = append(rsrcs, gvr.Resource)
+		rsrcs = append(rsrcs, gvr.Resource, "pods/ephemeralcontainers")
```

Comment:
@prateekpandey14 - won't `pods/ephemeralcontainers` be added in the webhook config regardless of matching kinds?

Reply:
Ping @prateekpandey14 .

Reply:
Yes,  I did added the contains check to verify pods kind first, if exists append the `pod/ephemeralcontainers`  as well. Which I yet to push.

Reply:
Ok let's add that in.

Reply:
done